src/pages/AdminManageUsers.js

```diff
@@ -125,7 +125,6 @@ export const AdminManageUsers = function AdminManageUsers() {
             className: 'btn-primary btn-add common-background',
             onClick: addUser
           }, [
-            div({ className: 'all-icons add-user_white' }),
             span({}, ['Add User']),
```

Comment:
minor optional nit: text in button is slightly off-center 
![Screenshot (93)](https://user-images.githubusercontent.com/81024249/185158277-3d962a7c-d6e0-415f-8ee5-d3ce497a0847.png)

Reply:
Quazi's covering the Institution fix on https://github.com/DataBiosphere/duos-ui/pull/1751 since he's already moving the button's position. This way there won't be any merge conflicts with his PR against mine.

Reply:
Gotcha, makes sense!